Extensions/titles.js

```diff
@@ -0,0 +1,104 @@
+//* TITLE Tab titles **//
+//* VERSION 1.0.0 **//
+//* DESCRIPTION Descriptive tab titles, rather than just “Tumblr” **//
+//* DEVELOPER Rebecca Turner **//
+//* FRAME false **//
+//* BETA false **//
+
+XKit.extensions.titles = new Object({
+	running: false,
+
+	/* object mapping the first url folder component to a function which takes
+	 * an array of url components (like ['blog', 'xkit'] for '/blog/xkit') and
+	 * returns the new page title
+	 */
+	titleFuncs: {
+		blog: url => {
+			if (url.length > 2) {
+				if (url[2] == "new") {
+					// new post
+					return "Post to " + url[1];
+				} else if (url[2] == "review") {
+					return url[1] + "’s flagged posts";
+				} else if (url[2] != "delete") {
+					// e.g. drafts, queue
+					return url[1] + "’s " + url[2];
+				} else {
+					return "Delete " + url[1];
+				}
+			}
+			// a plain blog page
+			return url[1];
+		},
+
+		explore: url => "Explore: " + {
+			"recommended-for-you": "Recommended for you",
+			"trending":            "Trending",
```

Comment:
object literal syntax should omit the `"` where possible, and because the majority of the keys here are so short and a few are very long, let's just make have consistent spacing instead of trying to align the values.

Reply:
Removed quotes, un-aligned values